free-courses-en.md

```diff
@@ -557,6 +557,7 @@
 
 * [Ethical Hacking](https://www.hacker101.com/videos) -(Hacker101)
 * [The Complete Cyber Security & Hacking Course](https://academy.ehacking.net/p/cyber-security-training-hacking-course) -(Insec Techs- EH Academy)
+* [Zero to hero network pentesting](https://www.thecybermentor.com/zero-to-hero-pentesting) -(The Cyber Mentor)
```

Comment:
This resource basically just links to YouTube anyway, so it could be nicer to go directly to the YouTube playlist instead?
It's better if we use the exact title of the resource to minimize potential for duplicates too.

```suggestion
* [Zero to Hero: A Practical Network Penetration Testing Course](https://www.youtube.com/playlist?list=PLLKT__MCUeiwBa7d7F_vN1GUwz_2TmVQj) - The Cyber Mentor
```